components/web/apps/org.wso2.carbon.apimgt.store/pom.xml

```diff
@@ -55,6 +55,12 @@
             <groupId>org.wso2.carbon.apimgt</groupId>
             <artifactId>org.wso2.carbon.apimgt.authenticator</artifactId>
         </dependency>
+        <dependency>
```

Comment:
This is not publisher app dependency. This is a dependency to publisher common component. We take API resource UI implementation from this component without duplicating the implementation.